settings.gradle

```diff
@@ -134,4 +137,14 @@ include 'instrumentation:jdbc-inet-merlia'
 include 'instrumentation:jdbc-inet-oranxo'
 include 'instrumentation:jdbc-sybase-6'
 include 'instrumentation:low-priority-instrumentation'
-
+include 'instrumentation:apache-struts2'
+include 'instrumentation:jersey'
+include 'instrumentation:resteasy-2.2'
+include 'instrumentation:resteasy-3'
+include 'instrumentation:resteasy-4'
+include 'instrumentation:apache-wicket-6.4'
+include 'instrumentation:apache-wicket-7.0'
+include 'instrumentation:apache-wicket-8.0'
+include 'instrumentation:grails-1.3'
+include 'instrumentation:grails-2.0'
+include 'instrumentation:grails-3.0'
```

Comment:
The above mention modules are missing from code base in this PR, hence the build will fail. Please remove them.

Reply:
Removed unnecessary module names in commit [0add19a](https://github.com/newrelic/csec-java-agent/pull/51/commits/0add19a2c8698f3de7b6613647780482163d7e5b)